File: app/src/main/java/com/akshayltc/chime/activities/SplashActivity.java
package com.akshayltc.chime.activities;

import android.os.Bundle;
import android.os.Handler;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;

import com.akshayltc.chime.R;

/**
 * Created by Admin on 2017-07-05.
 */

public class SplashActivity extends BaseActivity {

    //region global variables

    //endregion

    //region Application overrides
    final Handler handler = new Handler();

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);
        handler.postDelayed(new Runnable() {
            @Override
            public void run() {
                startFileBrowser();
                finish();
            }
        }, 200);
    }


    //endregion

}

File: app/src/main/java/com/whitelotusapps/chime/activities/SplashActivity.kt
package com.whitelotusapps.chime.activities

import android.os.Bundle
import android.os.Handler
import com.whitelotusapps.chime.R

/**
 * Created by Admin on 2017-07-05.
 */
class SplashActivity : BaseActivity() {
    //region global variables
    //endregion
    //region Application overrides
    val handler = Handler()
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)
        handler.postDelayed({
            startFileBrowser()
            finish()
        }, 200)
    } //endregion
}